src/index.js

```diff
@@ -14,11 +14,15 @@ const GitUtils = require('./GitUtils.js');
 const HelixConfig = require('./HelixConfig.js');
 const Strain = require('./Strain.js');
 const Logger = require('./Logger.js');
+const sequence = require('./sequence.js');
+const dom = require('./dom.js');
 
 module.exports = {
   GitUrl,
   GitUtils,
   HelixConfig,
   Strain,
   Logger,
+  ...sequence,
+  ...dom,
```

Comment:
same here. I would prefer some thing like
```suggestion
  DomUtils: dom,
```